### ISSUE TITLE
search at definitelytyped.org does not work correctly

### ISSUE THREAD

**Issue by chicoxyzzy** [DefinitelyTyped/tsd#164]
```bash
git:(master) ✗ tsd query eventemitter3

 - eventemitter3 / eventemitter3
```
but if you'll try to find it using http://definitelytyped.org/tsd/ it finds no results

**Comment by Diullei** [comment]
Hi @chicoxyzzy, thank you for reporting this issue. I'll verify....

**Comment by Diullei** [comment]
Fixed! #165
